inventory/host_vars/_example.com/secrets.example.yml

```diff
@@ -94,3 +94,6 @@ smtp_password:
 #active_record_encryption_primary_key: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
 #active_record_encryption_deterministic_key: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
 #active_record_encryption_key_derivation_salt: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+# VINE API settings
+#vine_api_url: "https://vine-url.com/api/v1"
```

Comment:
Do you think this would be ok to be a public var, defined in inventory (eg in host_vars/staging.*/config.yml)?

It should be easier to manage in the long run.

Reply:
Fixed now